civihr_default_theme/scss/base/_variables.scss

```diff
@@ -723,6 +723,9 @@ $container-large-desktop:      ((1140px + $grid-gutter-width)) !default;
 $container-lg:                 $container-large-desktop !default;
 
 
+$crm-wizard-font-color: #312D4B;
```

Comment:
where are we getting these colors from? I don't seem to find them in the [shoreditch's wizard](https://github.com/civicrm/org.civicrm.shoreditch/blob/926982a914cf6482562626219a71a434e34f4d79/scss/civicrm/common/_wizard.scss)

Reply:
These colors are new and mentioned in the mockup(https://projects.invisionapp.com/d/main#/console/10848582/229926107/preview), specific for SSP.